try{
console.log("tried",)
console.log("completed",)
} catch (error){
console.log("catched",)
console.log(error,)
}
